src/sql/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -59,6 +59,9 @@ export class SelectBox extends vsSelectBox {
 			this._selectedOption = newInput.selected;
 		}));
 
+		this.selectBackground = defaultStyles.selectBackground;
```

Comment:
whats going on here? you shouldn't need theres changes correct? maybe you need to do a master sync?

Reply:
The selectbox wasn't getting grayed out on disable anymore after pulling the latest bits. It might be a case of our selectbox implementation doing too much stuff, though. I'll take another look.

Reply:
Personally I would prefer isolating this change set to the edit data query stuff, since it will already be so large. Instead just make a bug and make it a separate change set